schemathesis_hooks.py

```
@@ -0,0 +1,170 @@
+                    argument.value.value = random.choice(uuids)
+        if node.name.value == "updateUser":
+            for argument in node.arguments:
+                if argument.name.value == "id":
+                    random.seed()
+                    argument.value.value = random.choice(uuids)
+        if node.name.value == "deleteUser":
+            for argument in node.arguments:
+                if argument.name.value == "id":
+                    argument.value.value = uuids.pop()
+        return body
+    except AttributeError as e:
```

Ruff (0.8.0)

Local variable `e` is assigned to but never used

Remove assignment to unused variable `e`

(F841)